pages/portfolio.html

```diff
@@ -4,9 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/portfolio.css">
+  <!--CSS loads on Chrome and Firefox, but not Safari-->
+  <script src="https://kit.fontawesome.com/cbd5459603.js" crossorigin="anonymous"></script>
+  <title>Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <h1 id="name">Megan Korling</h1>
+      <ul class="menu">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <h2>Software Development Portfolio</h2>
+    <article class="container">
+      <h3>Solar System Project</h3>
+      <p>This was a pair programming project where we created an API 
+        that connects with a postgres database to store, retrieve, update, and 
+        delete planet information. This was developed in Python
+        using Flask. It is deployed via Heroku.
+      </p>
+      <img src="../images/solar-system.png" alt="Preview of Solar System code">
```

Comment:
Great job with making sure all your images have alt text! We heart accessible websites. 💕